transcom/mymove
# INT-B-19779 - Show MTO Service Price for Counseling and Move Management

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19779)

## Summary

This branch now shows the price for approved Move Management (ReService Code "MS") and Counseling (ReService Code "CS) service items on the Move Task Order tab.


### How to test

1. Create a move then as SC send the move to prime by approving for Counseling/Move management.
2. As the Prime, access any move and create a payment request with Move Management and/or Counseling service items.
3. Login as a TOO and access the move, go to the "Move Task Order" tab.
4. At the bottom you should see a price like this:
![Screenshot 2024-06-21 at 12 59 34 PM](https://github.com/transcom/mymove/assets/146959895/67ff07c2-40a7-45f0-a198-f7ac35b63ddd)

Note: that this shows up before approving the item, you should see the same price on the "payment requests" page:
![Screenshot 2024-06-21 at 12 59 46 PM](https://github.com/transcom/mymove/assets/146959895/f0e14674-72af-45f3-b9bb-8dbcd826febd)

Not sure if this is intended behavior, but the routing functionality that actually shows these items under "approved service items" was implemented in E-04919 by Scrummy Bears and went through POA approval, so if this is a bug I believe it would be out of scope for this story.
